tools/ReleaseTools.psm1

```diff
@@ -322,11 +328,62 @@ function Update-Version {
         }
     }
 
+    Update-Branch -Version $Version
+
     if ($PSCmdlet.ShouldProcess("$RepositoryName/v$Version", "git commit")) {
-        Update-Branch -Version $Version
         git commit -m "Bump version to ``v$Version``"
+    } # TODO: Git reset to unstage
+
+    New-ReleasePR -RepositoryName $RepositoryName
+
+    Pop-Location
+}
+
+<#
+.SYNOPSIS
+  Creates a new draft GitHub PR from the release branch.
+.DESCRIPTION
+  Pushes the release branch to `origin` and then opens a draft PR.
+#>
+function New-ReleasePR {
+    [CmdletBinding(SupportsShouldProcess)]
+    param(
+        [Parameter(Mandatory)]
+        [ValidateSet([RepoNames])]
+        [string]$RepositoryName
+    )
+    # NOTE: This a side effect neccesary for Git operations to work.
+    Push-Location -Path "$PSScriptRoot/../../$RepositoryName"
+
+    $Version = Get-Version -RepositoryName $RepositoryName
+    $Branch = "release/v$Version"
+
+    Update-Branch -Version $Version
+
+    if ($PSCmdlet.ShouldProcess("$RepositoryName/$Branch", "git push")) {
+        Write-Output "Pushing branch ``$Branch``..."
+        git push origin $Branch
```

Comment:
I thought we had a nice invocation checker for this so that if the command fails we can throw

Reply:
Not that I know of in this module.

Reply:
The build module has something, but I think it's too much for this script. It's no huge deal if it fails, it'll just lead to the PR not being opened and a failure from there.